inlong-tubemq/tubemq-manager/src/main/java/org/apache/inlong/tubemq/manager/controller/cluster/ClusterController.java

```diff
@@ -132,9 +136,9 @@ public TubeMQResult queryCluster(@RequestParam(required = false) Integer cluster
         }
 
         MasterEntry masterNode = masterService.getMasterNode(clusterEntry.getClusterId());
-
+        Map<String, Integer> allCount = getAllCount(clusterId);
```

Comment:
use map as return value is inappropriate

Reply:
Map is not used here, what is better to return？